packages/app/src/h5web/App.module.css

```diff
@@ -1,5 +1,101 @@
 .root {
```

Comment:
This is where a lot of the global styles (previously in `base.css`), as well as all the variables end up. The main reason for the move is to nest styles and variables under `.root` so they don't apply to other things (notably in JupyterLab).